pkg/resource_quota/resource_quota.go

```diff
@@ -0,0 +1,42 @@
+package resourcequota
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+type ResourceQuota struct {
+	ID        string
```

Comment:
Whenever we show the ID, I think we should add the HREF field as well. In this case, I don't think the ID is needed. Let's remove it.